source/common/upstream/health_checker_impl.h

```diff
@@ -206,11 +206,11 @@ class HttpHealthCheckerImpl : public HealthCheckerImplBase {
     Http::ResponseHeaderMapPtr response_headers_;
     Buffer::InstancePtr response_body_;
     const std::string& hostname_;
-    const Http::Protocol protocol_;
     Network::ConnectionInfoProviderSharedPtr local_connection_info_provider_;
-    bool expect_reset_{};
-    bool reuse_connection_ = false;
-    bool request_in_flight_ = false;
+    const Http::Protocol protocol_;
```

Comment:
nit: please add a small note about the coalescing of these data members at the end (something like [this](https://github.com/envoyproxy/envoy/blob/1ac0a5948ca317fd418777dab2ffe27935c0a276/source/common/router/config_impl.h#L342)).

Reply:
Done